homeassistant/components/light/isy994.py

```diff
@@ -64,7 +64,7 @@ def turn_off(self, **kwargs) -> None:
         if not self._node.fastoff():
             _LOGGER.debug('Unable to turn on light.')
 
-    def turn_on(self, brightness=100, **kwargs) -> None:
+    def turn_on(self, brightness=255, **kwargs) -> None:
```

Comment:
Would you mind explaining what is happening here? I don't know enough to follow the logic.